content/blog/building-a-business-partnership-as-a-thriving-ecosystem.md

```diff
@@ -5,23 +5,23 @@ categories:
   - community-call
 authors:
   - moon-immisch
-image: /assets/images/elizabeth&myka
+image: /assets/images/elizabeth&myka.jpg
```

Comment:
& in urls is going to be tricky ... i would suggest correctly the `&` to something else like and.